cmd/fleet/main.go

```diff
@@ -478,10 +481,10 @@ LOOP:
 		ech := make(chan error, 2)
 
 		if started {
-			f.reporter.Status(proto.StateObserved_CONFIGURING, "Re-configuring", nil)
+			f.reporter.Status(proto.StateObserved_CONFIGURING, "Re-configuring", nil) //nolint:errcheck // unclear on what should we do if updating the status fails?
```

Comment:
maybe we log out the error on status update failure? having evidence of failure would help diagnose status desync with the daemon.